src/utils/getNextDateWithinBusinessHours/getNextDateWithinBusinessHours.ts

```diff
@@ -1,39 +1,74 @@
-import { formatISO, addDays, startOfDay, setHours, isValid } from 'date-fns'
+import { startOfDay, addDays, setHours, isValid, formatISO } from 'date-fns'
+import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz'
 
 const START_HOUR_WORKING_DAY = 9 // 9 AM
 const END_HOUR_WORKING_DAY = 17 // 5 PM
 
 type ISOString = string
 
+export const isDateBetweenBusinessHours = (
+  date: ISOString, // all dates in Awell are in UTC
```

Comment:
Using strings to represent dates is not a good practice. It's OK to do it when dates are included in serialised messages, but it's not the case here.
This function should accept a `Date` type instead of this `ISOString` type.